release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -108,6 +108,40 @@ For more information, see xref:../nodes/pods/nodes-pods-priority.html#non-preemp
 [id="ocp-4-7-monitoring"]
 === Monitoring
 
+[id="ocp-4-7-monitoring-prometheus-operator-alertmanagerconfig-crd-"]
+==== `AlertmanagerConfig` custom resource definition in Prometheus Operator not supported
+
+Modifying Alertmanager configurations by using the `AlertmanagerConfig` custom resource definition (CRD) in Prometheus Operator is not supported.
+
+For more information, see xref:../monitoring/configuring-the-monitoring-stack.html#support-considerations_configuring-the-monitoring-stack[Support considerations for monitoring].
+
+[id="ocp-4-7-monitoring-alerting-rule-changes"]
+==== Alerting rule changes
+
+{product-title} 4.7 includes the following alerting rule changes:
+
+* The `etcdBackendQuotaLowSpace` alert is added. The critical alert provides notification if the database size of an etcd cluster exceeds the defined quota on an etcd instance.
+* The `etcdHighFsyncDurations` alert is added. The critical alert provides notification if the 99th percentile fsync durations of an etcd cluster are too high.
+* The `etcdExcessiveDatabaseGrowth` alert is added. The warning alert provides notification if there is an observed surge in etcd writes that caused a 50% increase in database size on an etcd instance over a four-hour period.
+* The `NodeClockNotSynchronising` alert is updated ... // Need reason for update. I see that the `expr` changed to now include `and node_timex_maxerror_seconds >= 16`.
+* The `NodeNetworkReceiveErrs` alert is updated ... // Need reason for update. I see that the `expr` changed to `rate(node_network_receive_errs_total[2m]) / rate(node_network_receive_packets_total[2m]) > 0.01`.
```

Comment:
The reason behind the change is to make sure that the alert doesn't fire when only a handful number of errors is reported. Rather than using the absolute number of errors, the rule now uses the ratio of errors to total packets.